tests/test_wave_01.py

```diff
@@ -182,13 +182,14 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
We could also check that the movie we expect to be left in the watchlist list is still there, and that the other watched movie is properly in the watched list.